packages/react-native/ReactCommon/react/renderer/components/scrollview/ScrollViewShadowNode.cpp

```diff
@@ -66,7 +66,18 @@ void ScrollViewShadowNode::layout(LayoutContext layoutContext) {
 Point ScrollViewShadowNode::getContentOriginOffset() const {
   auto stateData = getStateData();
   auto contentOffset = stateData.contentOffset;
-  return {-contentOffset.x, -contentOffset.y + stateData.scrollAwayPaddingTop};
+
+  auto props = getConcreteProps();
```

Comment:
Apparently, you can't call this: CI fails with this error

```
/root/react-native/packages/react-native/ReactCommon/react/renderer/components/scrollview/ScrollViewShadowNode.cpp:70:8: error: call to implicitly-deleted copy constructor of 'ConcreteProps' (aka 'facebook::react::ScrollViewProps')
    auto props = getConcreteProps();
         ^       ~~~~~~~~~~~~~~~~~~
  /root/react-native/packages/react-native/ReactCommon/react/renderer/components/scrollview/ScrollViewProps.h:19:31: note: copy constructor of 'ScrollViewProps' is implicitly deleted because base class 'ViewProps' (aka 'facebook::react::HostPlatformViewProps') has a deleted copy constructor
  class ScrollViewProps final : public ViewProps {
                                ^
  /root/react-native/packages/react-native/ReactCommon/react/renderer/components/view/platform/android/react/renderer/components/view/HostPlatformViewProps.h:23:31: note: copy constructor of 'HostPlatformViewProps' is implicitly deleted because base class 'BaseViewProps' has a deleted copy constructor
  class HostPlatformViewProps : public BaseViewProps {
                                ^
  /root/react-native/packages/react-native/ReactCommon/react/renderer/components/view/BaseViewProps.h:23:23: note: copy constructor of 'BaseViewProps' is implicitly deleted because base class 'YogaStylableProps' has a deleted copy constructor
  class BaseViewProps : public YogaStylableProps, public AccessibilityProps {
                        ^
  /root/react-native/packages/react-native/ReactCommon/react/renderer/components/view/YogaStylableProps.h:18:27: note: copy constructor of 'YogaStylableProps' is implicitly deleted because base class 'Props' has a deleted copy constructor
  class YogaStylableProps : public Props {
                            ^
  /root/react-native/packages/react-native/ReactCommon/react/renderer/core/Props.h:39:3: note: 'Props' has been explicitly marked deleted here
    Props(const Props& other) = delete;
    ^
  1 error generated.
```

sooo... the fix is correct, but we need to find a way to safely get those `props`